src/cli_config.rs

```diff
@@ -48,6 +54,12 @@ pub fn cli_parse() -> CliConfig {
         .author(crate_authors!("\n"))
         .about(crate_description!())
         .setting(AppSettings::ColoredHelp)
+        .arg(Arg::with_name("udp-address")
+            .short("u")
+            .long("udp-address")
+            .help("UDP address to run DHT node")
+            .takes_value(true)
+            .default_value("0.0.0.0:33445"))
```

Comment:
Cannot be dehardcoded until some work is done to run the node both on IPv4 and IPv6

Reply:
What do you mean? If you specify here IPv4 address the node will wun in IPv4 mode. If you specify IPv6 address - IPv6 mode will be used.

Reply:
There should be a possibility to run both on IPv4 and IPv6.

Reply:
There is no such thing as both IPv4 and IPv6. When you run node in IPv6 mode you can send IPv4 packets using `::ffff:` prefix. And since it's IPv4 packet the recipient will see your IPv4 address and communicate with you as with normal IPv4 node.

Reply:
```
         // filtering ipv6 peer address is temporary fix,
         // dht_node may run as ipv4 only
         // or may run as having two socket (ipv4 socket and ipv6 socket)
        .filter(move |&(ref _packet, addr)| !(local_addr.is_ipv4() && addr.is_ipv6()))
```

Reply:
So the solution is not clear.

Reply:
```
        // filter out IPv6 packets if node is running in IPv4 mode
        .filter(move |&(ref _packet, addr)| !(udp_addr.is_ipv4() && addr.is_ipv6()))
```

Reply:
What is not clear?